en/docs/setup/data-purging.md

```diff
@@ -6,238 +6,28 @@ You can perform data purging by clearing the session data using the script given
     For more information about session persistence, see [Authentication Session Persistence](../../learn/authentication-session-persistence).
 
 !!! tip
-    It is recommended to run these steps at the time where server traffic is low. Especially if you are running this in the production environment for the first time, since the data volume to be purged may be higher. However, consider this as a housekeeping task that needs to be run at regular intervals. 
-    If you are doing this for the very first time on the production system begin with step 1. This is done as a best practice for introducing changes. Otherwise, you can skip the backup and verification steps, and schedule the aforementioned queries to be run at regular intervals.
+    It is recommended to run these steps at a time when server traffic is low. This recommendation is particularly important if you are running these steps in the production environment for the first time because the data volume to be purged may be higher. However, consider this as a housekeeping task that needs to be run at regular intervals. 
+    You can schedule the aforementioned queries to be run at regular intervals.
     
 
-1.  Take a backup of the running database.
-2.  Set up the database dump in a test environment and test it for any
-    issues. For more information on setting up a database dump, go to
-    the
-    [MySQL](https://dev.mysql.com/doc/refman/5.7/en/mysqldump.html#mysqldump-syntax)
-   , [SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/backup-restore/create-a-full-database-backup-sql-server)
-   , and
-    [Oracle](https://docs.oracle.com/cd/E11882_01/backup.112/e10642/rcmbckba.htm#BRADV8138)
-    offical documentation.
+1. Disable the internal session cleanup process by configuring the following properties in the `deployment.toml` file (which is stored in the `<IS_HOME>/repository/conf` folder).
 
-    !!! tip
-        We recommend that you test the database dump before the cleanup task as the cleanup can take some time.
+    ```toml
+    [session_data.cleanup]
+    enable_expired_data_cleanup = false
+    clean_logged_out_sessions_at_immediate_cycle = false
+    enable_pre_session_data_cleanup = false
     
+    ```
 
-3.  To clean the session and operation data that are stored in the `IDN_AUTH_SESSION_STORE` table, run the following script on the database dump. 
+2. To clean the session and operation data that are stored in the `IDN_AUTH_SESSION_STORE` table, run the required database script on the database based on the database type. 
 
     !!! note
         This script clears session data that are older than the last 14 days and operational data that are older than the last 6 hours.
 
-    ??? tip "Click to view the MySQL script" 
```

Comment:
Is it necessary to remove this? This seems to be an example for MySQL.
@deshankoswatte

Reply:
Okay. I think I found the answer. Since we have pointed to all the DB scripts in the below note, I suppose this is redundant. +1